chia/harvester/harvester_api.py

```diff
@@ -114,7 +114,20 @@ def blocking_lookup(filename: Path, plot_info: PlotInfo) -> List[Tuple[bytes32,
                             # Found a very good proof of space! will fetch the whole proof from disk,
                             # then send to farmer
                             try:
+                                full_proof_start = time.time()
                                 proof_xs = plot_info.prover.get_full_proof(sp_challenge_hash, index)
+                                full_proof_duration = time.time() - full_proof_start
+
+                                if full_proof_duration > 10:
```

Comment:
Why 10? I thought timelords will honor it until 30s, also someone said check needs to happen within 30s but proof within 1min. Do you have a source for this 10s?

Reply:
Fair point, thanks! (it's the other PR that is changing the read pattern, but yes, it's generally very good)

by the way shouldn't it be 6 sequential seeks since we already have the root node (from the quality check)?

Reply:
Yeah, you're right! Also, depending on how smart your OS is at caching, some of the nodes are already cached due to the quality read which accessed the same values.

This backs up my experience that with my optimized read pattern, the full proof is actually faster than the quality retrieval. So if you surpass the 10 seconds, something must be oddly wrong here